class UserNotFoundException(Exception):
    pass


class EmailTakenException(Exception):
    pass
